packages/editor-ui/src/views/SettingsUsersView.vue

```diff
@@ -9,7 +9,11 @@
 					</template>
 					<div>
 						<n8n-button
-							:disabled="ssoStore.isSamlLoginEnabled"
+							:disabled="
```

Comment:
Ran out of time, self reminder to keep reviewing later from here.

Reply:
Remove all the `isUserManagementEnabled` checks, relying on quota